console/src/components/Common/utils/v1QueryUtils.ts

```diff
@@ -98,6 +98,65 @@ export const getDropPermissionQuery = (
   };
 };
 
+export const getInsertUpQuery = (
+  tableDef: TableDefinition,
+  insertion: object
+) => {
+  return {
+    type: 'insert',
+    args: {
+      table: tableDef,
+      returning: [],
+      object: insertion,
+    },
+  };
+};
+
+interface ColumnType {
+  column_name: string;
+}
+
+type PrimaryKeyInfo = {
```

Comment:
Could you move this type to `pgUtils.ts`, and add an adequate field to `Table` types?